endpoint/endpoint_diagnoseComp/endpoint_diagnose.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2019-2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+
+#ifndef ENDPOINT_DIAGNOSE_H
+#define ENDPOINT_DIAGNOSE_H
```

Comment:
Why did you create such an empty header?

Reply:
For the function declaration?

Reply:
Last time I checked this file, there was no function prototype.

Reply:
It did not have the function prototype last time.
And since the timer is added, I need an extra function.
Hence, I add the function prototype this time.